usagereportsv4/usage_reports_v4.go

```diff
@@ -1029,6 +1512,90 @@ func (options *GetOrgUsageOptions) SetHeaders(param map[string]string) *GetOrgUs
 	return options
 }
 
+// GetReportsSnapshotConfigOptions : The GetReportsSnapshotConfig options.
+type GetReportsSnapshotConfigOptions struct {
+	// Account ID for which the billing report snapshot is configured.
+	AccountID *string `json:"account_id" validate:"required"`
+
+	// Allows users to set headers on API requests
+	Headers map[string]string
+}
+
+// NewGetReportsSnapshotConfigOptions : Instantiate GetReportsSnapshotConfigOptions
+func (*UsageReportsV4) NewGetReportsSnapshotConfigOptions(accountID string) *GetReportsSnapshotConfigOptions {
+	return &GetReportsSnapshotConfigOptions{
+		AccountID: core.StringPtr(accountID),
+	}
+}
+
+// SetAccountID : Allow user to set AccountID
+func (_options *GetReportsSnapshotConfigOptions) SetAccountID(accountID string) *GetReportsSnapshotConfigOptions {
+	_options.AccountID = core.StringPtr(accountID)
+	return _options
+}
+
+// SetHeaders : Allow user to set Headers
+func (options *GetReportsSnapshotConfigOptions) SetHeaders(param map[string]string) *GetReportsSnapshotConfigOptions {
+	options.Headers = param
+	return options
+}
+
+// GetReportsSnapshotOptions : The GetReportsSnapshot options.
+type GetReportsSnapshotOptions struct {
+	// Account ID for which the billing report snapshot is requested.
+	AccountID *string `json:"account_id" validate:"required"`
+
+	// The month for which billing report snapshot is requested.  Format is yyyy-mm.
+	Month *string `json:"month" validate:"required"`
+
+	// Timestamp in milliseconds for which billing report snapshot is requested.
+	DateFrom *float64 `json:"date_from,omitempty"`
```

Comment:
this should probably be of type `*int64`.  Are you sure that the API definition is correct and agrees with the server implementation?